### ISSUE TITLE
Is there a bug in the asynchronous reasoning of smolvla?

### ISSUE THREAD

**Issue by w646** [huggingface/lerobot#1784]
Thank you very much for your open-source work, but there may be an issue with the asynchronous inference in SmolVLA: the _load_as_safetensor function is overridden in SmolVLAPolicy, and the normalization and unnormalization related parameters are ignored from the checkpoint. This will result in the normalize and unnormalize parameters being set to inf during from_pretrained. Of course, normalize and unnormalize can also be initialized in the SmolVLAPolicy's init via dataset_stats, but dataset_stats is not passed during asynchronous inference, which could lead to runtime errors, right?

**Comment by godnpeter** [comment]
I'm experiencing this issue. 
How are we suppose to resolve this?
Are we suppose to pass on the task training data statistics in order to perform eval?

**Comment by pkooij** [comment]
@fracapuano

**Comment by fracapuano** [comment]
(wanted to add that this issue is not due to async inference and it is likely fixed by the preprocessors pipeline / more stable instantiation of pretrained policy models)
